packages/ecl/build.sh

```diff
@@ -2,7 +2,6 @@ TERMUX_PKG_HOMEPAGE=https://common-lisp.net/project/ecl/
 TERMUX_PKG_DESCRIPTION="ECL (Embeddable Common Lisp) is an interpreter of the Common Lisp language."
 TERMUX_PKG_LICENSE="LGPL-2.0"
```

Comment:
fwiw ecl is licensed under lgpl-2.1+, not lgpl-2.0

Reply:
I put LGPL-2.0 here because it says "Version 2" in https://gitlab.com/embeddable-common-lisp/ecl/-/blob/develop/COPYING

Reply:
you are right, the included license is outdated (we already
bundle lgpl-2.1+ sources of other libraries)

 (I will also need to change that), either way, since there is

```
    This library is free software; you can redistribute it and/or
    modify it under the terms of the GNU Library General Public
    License as published by the Free Software Foundation; either
    version 2 of the License, or (at your option) any later version.
```

it should be at least mentioned as lgpl-2.0+